device/device.yaml

```diff
@@ -68,18 +68,18 @@ components:
           $ref: './rsvp/rsvp.yaml#/components/schemas/Device.Rsvp'
           x-field-uid: 8
       required: [name]
-    Device.Options:
+    Protocol.Options:
       description: >-
-        Common options that apply to all configured devices. 
+        Common options that apply to all configured protocols and interfaces. 
       type: object
       properties:
         auto_start_all:
           description: >-
-            When set to true, all underlying resources for configured devices 
-            shall be created and corresponding protocol session negotiation shall 
-            be initiated. Otherwise, when set to false, corresponding protocol 
-            session negotiation will need to be initiated using a separate 
-            set_protocol_state API call.
+            When set to true, all underlying resources for configured protocols  
```

Comment:
I understand that when L47 state-full flows become part of OTG, then user will specifically need to do something similar to StartTransmit explicitly and this will not in ambit of auto_start_all=true option.